src/Extensions/CwpSiteSummaryExtension.php

```diff
@@ -3,12 +3,24 @@
 namespace CWP\CWP\Extensions;
 
 use SilverStripe\Core\Extension;
+use SilverStripe\Dev\Deprecation;
 
 /**
  * Extends the site summary report to list the appropriate versions in the report header
+ *
+ * @deprecated 4.13.0 Will be removed without equivalent functionality to replace it
  */
 class CwpSiteSummaryExtension extends Extension
 {
+    public function __construct()
+    {
+        Deprecation::notice(
```

Comment:
Should be wrapped in `withNoReplacement()` since we're still using this in CMS 4.

Reply:
Updated

Reply:
Heads up, this just created an error for me when using with `silverstripe/framework@4.12.3`
> Error: Call to undefined method SilverStripe\\Dev\\Deprecation::withNoReplacement()

([ref](https://github.com/silverstripe/silverstripe-framework/blob/4.12.3/src/Dev/Deprecation.php))

Reply:
@edwilde Thanks, I'll update the dependency for `2.11` of this module.